### PR TITLE
ShowDeviceTrackingDatabase

### PR DESCRIPTION
**Pull request by itdependsnetworks**

### CODE REVIEW

File: src/genie/libs/parser/iosxe/show_device_tracking.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        # 0001:MAC and LLA match     0002:Orig trunk            0004:Orig access
+        pref_level_flags_2_capture = re.compile(
+            r"^0001:MAC\s+and\s+LLA\s+match\s+0002:Orig\s+trunk\s+0004:Orig\s+access$")
+        # 0008:Orig trusted trunk    0010:Orig trusted access   0020:DHCP assigned
+        _capture = re.compile(r"^0008:Orig\s+trusted\s+trunk\s+0010:Orig\s+trusted\s+access\s+0020:DHCP\s+assigned$")
+        # 0040:Cga authenticated     0080:Cert authenticated    0100:Statically assigned
+        _capture = re.compile(r"^0040:Cga\s+authenticated\s+0080:Cert\s+authenticated\s+0100:Statically\s+assigned$")
+        #     Network Layer Address                   Link Layer Address Interface  vlan  prlvl age    state     Time left
+        device_info_header_capture = re.compile(
+            r"^\s+Network\s+Layer\s+Address\s+Link\s+Layer\s+Address\s+Interface\s+vlan\s+prlvl\s+age\s+state\s+Time\s+left$")
```
Comment (LenzDong) [inline src/genie/libs/parser/iosxe/show_device_tracking.py:87]: You will trim the space for each line, so you can remove the first ```\s+```  like ```r"^Network\s+Layer

File: src/genie/libs/parser/iosxe/show_device_tracking.py [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+        # L   10.22.66.10                            7081.0535.b60b     Vl230      230   0100  10194mn REACHABLE
+        device_info_capture = re.compile(
+            r"^(?P<dev_code>\S+)\s+(?P<network_layer_address>\d+\.\d+\.\d+\.\d+)\s+(?P<link_layer_address>\S+)\s+(?P<interface>\S+)\s+(?P<vlan_id>\d+)\s+(?P<pref_level_code>\d+)\s+(?P<age>\S+)\s+(?P<state>\S+)$")
```
Comment (LenzDong) [inline src/genie/libs/parser/iosxe/show_device_tracking.py:90]: Is it possible to have ipv6? maybe change to ```(?P<network_layer_address>\S+)``` is safer

File: src/genie/libs/parser/iosxe/show_device_tracking.py (hunk 1 above)
Comment (qduk) [reply src/genie/libs/parser/iosxe/show_device_tracking.py:87]: Updated

File: src/genie/libs/parser/iosxe/show_device_tracking.py (hunk 2 above)
Comment (qduk) [reply src/genie/libs/parser/iosxe/show_device_tracking.py]: Yes, it looks like it is possible to have ipv6 entries. Updated regex to `\S+`

**Review by LenzDong — APPROVED** [review]

**Review by ThomasJRyan — APPROVED** [review]

**Pull request merged**